### DISCUSSION TITLE
Is console support planned?

### DISCUSSION THREAD

**Discussion by freeeranger** [DiligentGraphics/DiligentEngine#126] [General]
Hey, so I'm going to make my own game engine and I'm very interested in Diligent Engine. One thing that is quite unfortunate is that Diligent Engine doesn't have console support while BGFX (the other library that I'm considering) does. So I'm basically just wondering if this is something that is planned or if it will be added in the future.

**Comment by TheMostDiligent** [comment]
Hello!
We have not had immediate plans to enable console support so far.

**Comment by TheMostDiligent** [comment]
Note that XBOX allows running UWP application, so as since Diligent supports UWP, it technically supports Xbox.

**Reply by freeeranger** [reply]
Ok, that's good to know. Do you think that consoles might be supported in the future (like in 3+ years or smthn)? I understand if you can't answer the question with 100%  or even 50% certainty, but I'd just like to know if it's possible that it gets added in the future. Thanks!

**Reply by TheMostDiligent** [reply]
Yes, it is certainly possible if there is an interest. So far you are the first who asked about the console support.

**Reply by freeeranger** [reply]
Alright, that's good to hear - I probably won't need it for another 5 years or smthn so no probs there. Let's just hope more people show interest in console support then :D

**Comment by Qubrick** [comment]
Is there any console support yet?

I think it's ok if you make it as a commercial license similar to Metal.

**Reply by TheMostDiligent** [reply]
The status is the same - XBox should work, there is no PS support. We are not planning PS support as of today.
